### PR TITLE
Fix: Resolve and clean up compile errors

### PR DESCRIPTION
**Pull request by CodeMeKathy**
## Description
This Pull Request (PR) resolves and cleans up compile errors while also correcting the JSON parse path to resolve the `profile` blank page error.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|   ✅   | :bug: Bug fix(es)              |
|    | :sparkles: New feature     |
| ✅    | :hammer: Refactoring       |
|     | :100: Add tests            |
|    | :link: Update dependencies |
|   ✅  | :scroll: Docs              |

### CODE REVIEW

**Review by sergiobtos — APPROVED** [review]

**Review by sergiobtos — COMMENTED** [review]
Why are you deleting ".data"? whithout this does not work because the profile is saved on local storage like using the key work "data"

{"data":{"result":{"name":"Sergio Silva","email":"bsergiobtos@hotmail.com","password":"$2a$12$Y35Oq.XEO.SPNk2aeiM2.ePFfb8fpQgJiOcPEOD2RT7hWKV6ooqbS","role":"owner","createdAt":"2022-05-21T00:43:47.284Z","imageProfile":"data:.....

" Hi, @sergiobtos - Per our team meeting we confirm accessing the profile data object via local storage does not require `.data` within the path. See the enclosed image for additional details.

Thanks!

~ Kathy"

<details>
<summary>Broswer Profile Data Object</summary>

Broswer Profile Data Object Image



</details>

File: client/src/components/MyBusinessList/MyBusinessList.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import '../../pages/Profile/Profile.css'
 export const MyBusinessList = () => {
   const [list, setList] = useState<any[]>([])
   const [loading, setLoading] = useState(true)
-  const user = JSON.parse(localStorage.getItem('profile') || '{}').data.result; 
+  const user = JSON.parse(localStorage.getItem('profile') || '{}').result;
```
Comment (CodeMeKathy) [inline client/src/components/MyBusinessList/MyBusinessList.tsx:14]: Hi @sergiobtos!  Thank you for your review.  In response to your [comment](https://github.com/Brridge-Group/Lions-Gate-Hairr/pull/34#pullrequestreview-980800667).  I have modified this line of code due to the associated terminal error: `Uncaught TypeError: can't access property "result," JSON.parse(...).data is undefined`.  See enclosed image.  After debugging, I located the data elsewhere.  The modification restored the blank profile page to the intended profile page UI.  I also conferred with @cynthiaeddy, who, I believe, initially wrote this line of code, that is connected with a line of code in the <Profile/> component.

Terminal Error Image

File: client/src/pages/Profile/Profile.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { MyBusinessList } from '../../components/MyBusinessList/MyBusinessList'
 import './Profile.css'
 
 export const Profile = () => {
-  const {role, _id, name, imageProfile} = JSON.parse(localStorage.getItem('profile') || '{}').data.result; 
+  const {role, _id, name, imageProfile} = JSON.parse(localStorage.getItem('profile') || '{}').result;
```
Comment (CodeMeKathy) [inline client/src/pages/Profile/Profile.tsx:8]: Hi @sergiobtos!  Thank you for your review.  In response to your [comment](https://github.com/Brridge-Group/Lions-Gate-Hairr/pull/34#pullrequestreview-980800667).  I have modified this line of code due to the associated terminal error: `Uncaught TypeError: can't access property "result," JSON.parse(...).data is undefined`.  See enclosed image.  After debugging, I located the data elsewhere.  The modification restored the blank profile page to the intended profile page UI.  I also conferred with @cynthiaeddy, who, I believe, initially wrote this line of code, that is connected with a line of code in the <Profile/> component.

Terminal Error Image

**Review by sergiobtos — APPROVED** [review]

**Review by sergiobtos — APPROVED** [review]

**Review by cynthiaeddy — APPROVED** [review]
great as always, you're da best.

**Pull request merged**